src/components/AnonymousChat/child/ChannelFollowButton.tsx

```diff
@@ -3,8 +3,8 @@ import React from 'react';
 import {Pressable, StyleSheet, Text, View} from 'react-native';
 
 import dimen from '../../../utils/dimen';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the new `COLORS` object from the `theme` module instead of the old `colors` module. This change is consistent with the PR description and seems to be part of a larger refactoring effort to centralize color management in the application.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->